paint-timing/fcp-only/fcp-viewport.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<head>
+<title>Performance Paint Timing Test: FCP with element far from viewport</title>
```

Comment:
I'm not sure this test is valid, as a user agent could choose not to paint content so far away from the viewport. And then FCP would not trigger.

Reply:
right, will remove for now.